### PR TITLE
feat:card component added

### PR DESCRIPTION
**Pull request by sharmayatendra**
suggestions are welcome please review the code

### CODE REVIEW

File: components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
+:root {
+    --card-hover-color: rgb(243, 241, 241);
+    --card-text-color: rgb(128, 128, 128);
+    --bg-color: rgb(255, 255, 255);
+    --bg-color2: rgb(56, 142, 60);
```
Comment (xii-vi) [inline components/Card/card.css]: Use a different name for two same variables instead of number.

File: components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    border-radius: 2px;
+    background-color: var(--badge-color);
+    color: var(--bg-color);
+    position: absolute;
+    left: 0rem;
+    top: 1rem;
+    display: inline-block;
+}
+
+.img-rel {
```
Comment (xii-vi) [inline components/Card/card.css:124]: Make a utility class for these position thingy

File: components/Card/card.html [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+            <p class="main-description">Cards are used to showcase the information regarding a product, text etc. so
+                that user can easily see it.
+            </p>
+            <p class="main-description"><span>Amaze-UI</span> has different kinds of cards.</p>
+            <div class="comp-container">
+                <h2 class="comp-heading">Vertical Card</h2>
+                <div class="card">
+                    <div class="card-wrapper">
+                        <div class="card-img">
+                            <img src="/images/OnePlus.jpg" alt="river-scene">
```
Comment (xii-vi) [inline components/Card/card.html]: user proper alternative

**Review by xii-vi — COMMENTED** [review]
Work on the suggested comment.
Rest is good to go.

File: components/Card/card.css (hunk 1 above)
Comment (vikrantkode) [inline components/Card/card.css]: variable name should be better, personal choice.

Comment (sharmayatendra) [reply components/Card/card.css]: done

**Review by vikrantkode — CHANGES_REQUESTED** [review]
Looks good to me.
Some suggestion given.

File: components/Card/card.html (hunk 3 above)
Comment (sharmayatendra) [reply components/Card/card.html]: done

**Pull request merged**